### DISCUSSION TITLE
Error Platform x86

### DISCUSSION THREAD

**Discussion by QusaiImayyah** [QuestPDF/QuestPDF#902] [General]
![Screenshot 2024-05-22 161729](https://github.com/QuestPDF/QuestPDF/assets/20271178/8dcc6039-d3a9-46fa-b9fc-8deb3d6b996a)

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

The post only contained a screenshot without further details, so we were unable to investigate. If you are still experiencing this, please feel free to open a new issue with the full error message and your project setup so we can take a closer look.

Thank you!
